pages/common/gpg-tui.md

```diff
@@ -0,0 +1,36 @@
+# gpg-tui
+
+> Terminal user interface for GNU Public Guard.
+> More information: <https://github.com/orhun/gpg-tui>.
+
+- Start `gpg-tui`:
+
+`gpg-tui`
+
+- Start `gpg-tui` enabling color and ASCII armored output:
+
+`gpg-tui --style {{colored}} --armor`
+
+- Show help in `gpg-tui`:
+
+`?`
+
+- Quit `gpg-tui`:
+
+`q`
+
+- Interactively generate a new key:
+
+`g`
+
+- Export the selected key:
+
+`x`
+
+- Set the detail level for the selected key:
+
+`1|2|3`
+
+- Refresh `gpg-tui`:
```

Comment:
```suggestion
- Refresh the view:
```

Reply:
I can't really agree here. This doesn't just refresh the view, it basically restarts `gpg-tui`